core/server/api/v2/utils/serializers/input/settings.js

```diff
@@ -139,6 +139,19 @@ module.exports = {
                 setting.value = JSON.parse(setting.value).isActive;
             }
 
+            if (setting.key === 'labs') {
```

Comment:
Do we have a use case for allowing labs to be writable by any API version other than `canary`? Wondering if it makes sense to keep labs as a canary-only feature so there's less maintenance overhead

Reply:
That said, having the allowed keys list live in the service keeps maintenance minimal for older API versions 🤔

Reply:
This is an awesome point! "labs" is a synonym of "canary" and doesn't make sense to allow for it in the stable APIs. Additional upside - less code to maintain.

Reading is a different topic and totally makes sense to have labs available as readable property across all the clients.

The only reason why I can think of for leaving labs as a writable option is a convenience for the API client where they are pegged to a certain stable version and would need to reconfigure SDK (whatever way they access the API) with canary endpoint. But this is an edge case I think we can address IF there's anybody even complaining about it :thinking:

Reply:
Agreed I wouldn't expect any client other than admin to be writing to this setting.